### PR TITLE
adding replica set to mongo for the breaking change in v1.0

### PR DESCRIPTION
**Pull request by sshniro**
Fix for issue #80

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+    depends_on:
+      - mongo
+
   rocketchat:
     image: rocket.chat:latest
+    restart: unless-stopped
     environment:
-      ROOT_URL: 'http://localhost'
+      - ROOT_URL=http://localhost
+      - MONGO_URL=mongodb://mongo:27017/rocketchat
```
Comment (geekgonecrazy) [inline docker-compose.yml]:
```suggestion
      - MONGO_URL=mongodb://mongo:27017/rocketchat?replicaSet=rs0
```

File: docker-compose.yml [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+      - MONGO_OPLOG_URL=mongodb://mongo:27017/local
```
Comment (geekgonecrazy) [inline docker-compose.yml]:
```suggestion
      - MONGO_OPLOG_URL=mongodb://mongo:27017/local?replicaSet=rs0
```

**Comment by geekgonecrazy** [comment]
I think as long as consistent with all docs it’s ok

File: docker-compose.yml [hunk 3]
```diff
@@ -1,13 +1,25 @@
 version: '3.6'
 services:
-  db:
+  mongo:
     image: mongo:3.2
-    entrypoint: mongod --smallfiles
     volumes:
-      - ./data:/data/db
+      - ./data/db:/data/db
+    command: mongod --smallfiles --oplogSize 128 --replSet rs0
```
Comment (geekgonecrazy) [inline docker-compose.yml]:
```suggestion
    command: mongod --smallfiles --oplogSize 128 --replSet rs01
```

**Comment by sshniro** [comment]
@Sing-Li  & @geekgonecrazy  the suggestions have been incorporated and tested locally.

**Pull request merged**